### ISSUE TITLE
Full screen method

### ISSUE THREAD

**Issue by ruudboon** [jepiqueau/capacitor-video-player#29]
Would it be possible to add a fullscreen event? I would like to trigger fullscreen video from my app.
I can imagine this isn't technically possible but maybe it is :D

**Comment by jepiqueau** [comment]
@ruudboon Do you mean a fullscreen event? , which will return true or false

**Comment by ruudboon** [comment]
No, a method to trigger fullscreen from JS. I'm thinking about a page in my app that will display a countdown and small preview of a streaming video. On 0 I want to launch it fullscreen.

**Comment by jepiqueau** [comment]
@ruudboon Is it for Web platform or/and Native platforms? For the time being on Native is always fullscreen.

**Comment by ruudboon** [comment]
@jepiqueau Web app. But maybe I will switch to native. Fullscreen in more important then embed for me.

**Comment by jepiqueau** [comment]
@ruudboon Ok i will have a look but it will be only for web app.

**Comment by ruudboon** [comment]
@jepiqueau Thnx! appreciate it.

**Comment by jepiqueau** [comment]
@ruudboon is it something urgent? It will accessible through api. i will see the feasibility. It may be also interesting to double click on the embedded video to get it fullscreen

**Comment by ruudboon** [comment]
@jepiqueau No it's not urgent at this point. Gonna work this weekend on a proof of concept. Got some more boxes too thick. If it all pans out I probably need this functionality in a few months.
